### PR TITLE
Update fr.json

### PR DESCRIPTION
**Pull request by DariaPow**

### CODE REVIEW

**Comment by znarf** [comment]
@Betree can you review this?

File: src/lang/fr.json [hunk 1]
```diff
@@ -436,7 +436,7 @@
   "host.dashboard": "Dashboard",
   "menu.transactions": "transactions",
   "menu.team": "équipe",
-  "menu.createPledge": "Make a Pledge",
+  "menu.createPledge": "Faire un gage",
```
Comment (Betree) [inline src/lang/fr.json]:
```suggestion
  "menu.createPledge": "Faire une promesse de don",
```

File: src/lang/fr.json [hunk 2]
```diff
@@ -513,16 +513,16 @@
...
   "update.publish.btn": "publier",
   "update.publish.backers": "Votre billet sera envoyé à vos {n} contributeurs",
-  "redeem.form.code.label": "Gift card code",
-  "redeem.card.info": "It’s easy. Just provide your name and email address, enter your gift code and your open collective account will be created (if needed) and the gift card will be automatically attached to your account.",
-  "redeem.card.success.title": "You’re just one step away from\n    supporting open initiatives.",
-  "redeem.card.success.instructions": "We’ve sent an email to {email} with instructions to verify your new Open Collective account and credit it with the amount of your gift card.",
-  "paymentMethod.bitcoin.error.invalidEmail": "We can't generate a bitcoin address without a valid email address.",
+  "redeem.form.code.label": "Le code de carte-cadeau",
+  "redeem.card.info": "C’est facile. Il ne faut que donner votre nom, prénom et l’adresse mail, taper le code de votre carte-cadeau et votre compte de collectif sera créé (si nécessaire). Le carte-cadeau sera automatiquement attaché à votre compte.",
```
Comment (Betree) [inline src/lang/fr.json]:
```suggestion
  "redeem.card.info": "C’est facile. Entrez juste votre nom, prénom et l’adresse mail, puis tapez le code de votre carte-cadeau et votre compte de collectif sera créé (si nécessaire). La carte-cadeau sera automatiquement attachée à votre compte.",
```

File: src/lang/fr.json [hunk 3]
```diff
@@ -513,16 +513,16 @@
...
+  "redeem.card.success.title": "Vous n'êtes qu'à un pas pour soutenir des initiatives d'Open Collective!",
```
Comment (Betree) [inline src/lang/fr.json]:
```suggestion
  "redeem.card.success.title": "Vous n'êtes qu'à un pas de soutenir des initiatives d'Open Collective !",
```

File: src/lang/fr.json [hunk 4]
```diff
@@ -513,16 +513,16 @@
...
+  "redeem.card.success.instructions": "On a envoyé un e-mail à l'adresse {email} avec des instructions comment vérifier le nouveau compte de votre collectif et le montant de votre carte-cadeau.",
```
Comment (Betree) [inline src/lang/fr.json]:
```suggestion
  "redeem.card.success.instructions": "Nous avons envoyé un e-mail à l'adresse {email} avec des instructions pour vérifier le nouveau compte de votre collectif et créditer le montant de votre carte-cadeau.",
```

File: src/lang/fr.json [hunk 5]
```diff
@@ -561,20 +561,20 @@
   "signinup.description.label": "Rapide description",
   "signinup.description.description": "Présentez-vous en 60 caractères, ou moins!",
   "signinup.twitterHandle.label": "twitter",
-  "subscription.amountToDate": "contributed to date",
+  "subscription.amountToDate": "Ont contribué jusqu'à présent",
```
Comment (Betree) [inline src/lang/fr.json]:
```suggestion
  "subscription.amountToDate": "ont contribué jusqu'à présent",
```

File: src/lang/fr.json [hunk 6]
```diff
@@ -674,17 +674,17 @@
...
   "tier.name.payment": "paiement",
-  "updates.new.title": "New update",
+  "updates.new.title": "Une nouvelle mise à jour",
   "updates.create.login": "Vous devez vous identifier en tant qu'administrateur de ce collectif pour pouvoir créer un nouveau billet.",
   "expenses.byCategory": "Dépenses en {category}",
   "expenses.byRecipient": "Dépenses par {recipient}",
-  "error.code.invalid": "Invalid gift card code",
-  "redeem.subtitle.line1": "Open Collective helps communities - like open source projects, meetups, etc - raise money and operate transparently.",
+  "error.code.invalid": "Le code de carte-cadeau pas valable",
+  "redeem.subtitle.line1": "Open Collective aide des communautés - projets open-source, meetups etc - à collecter de l’argent et à opérer dans une façon transparente.",
```
Comment (Betree) [inline src/lang/fr.json]:
```suggestion
  "redeem.subtitle.line1": "Open Collective aide des communautés - projets open-source, meetups etc - à collecter de l’argent et à opérer de façon transparente.",
```

Comment (DariaPow) [reply src/lang/fr.json]: Hi Betree! I tried to commit all your suggestions (thank you so much for checking btw!) but I'm know sure if I did it right. Did it work ??

**Comment by Betree** [comment]
This is now merged, thank you again @DariaPow :pray:

**Comment by DariaPow** [comment]
@Betree no problem! I see that one change didn't come through, did you manage to push it through ? cause I cannot commit the change

**Comment by Betree** [comment]
We havn't deployed it yet to https://opencollective.com.

I think all of your changes were merged, which one do you think is missing?

**Comment by DariaPow** [comment]
I have a red cross next to your changes d06e90e and dcd230c. As this is my first time with GitHub, I'm not sure what does it mean but logically I have impression that those changes haven't been merged

**Comment by Betree** [comment]
Oh no don't worry, these are indicating that our code tests failed (actually they were not triggered because we did too much commits in a short time). But everything's fine, you can see what was actually merged in the "Files Changed" tab :slightly_smiling_face:

**Pull request merged**